test/Examples_Tests/data/signatures/Meta.md

```diff
@@ -0,0 +1,48 @@
+## Enso Signatures 1.0
+## module Standard.Base.Meta
```

Comment:
Snapshot of the `Meta` API used by the [Signature_Spec](https://github.com/enso-org/enso/pull/12031#discussion_r1923133269)